projects/libfido2/build.sh

```diff
@@ -61,6 +61,7 @@ for f in `ls fuzz/fuzz_*`; do
 done
 
  # Prepare seed corpora
+curl -s https://ambientworks.net/libfido2/corpus.tgz > ${SRC}/libfido2/fuzz/corpus.tgz
```

Comment:
I'd recommend moving this to `GET` command in Dockerfile. That way it'll execute only once, when creating a container, rather than being executed on every build

Reply:
thanks, makes sense, will update the PR